doc/src/manual/types.md

```diff
@@ -1153,7 +1153,7 @@ julia> typeof(Rational{Int})
 DataType
 
 julia> typeof(Union{Real,Float64,Rational})
-DataType
+Union
```

Comment:
I get `DataType` though?
```
julia> typeof(Union{Real,Float64,Rational})
DataType
```

Reply:
Strange, so do I now. Maybe I didn't try it exactly as written :(

Anyhow, removed entirely at @yuyichao's suggestion.